controllers/admin/AdminEmailsController.php

```diff
@@ -235,7 +235,12 @@ public function setMedia()
 	
 	public function processDelete()
 	{
-		return Mail::eraseAllLogs();
+		if ((int)$id_mail = Tools::getValue('id_mail', 0))
```

Comment:
Hi,

If you set a default value for Tools::getValue() it will always enter in the first conditions and never do a Mail::eraseAllLogs();